tcms/testcases/templates/testcases/mutable.html

```diff
@@ -100,6 +100,25 @@
                 </div>
             </div>
 
+            <div class="form-group">
+                <div>
+                    <label class="col-md-1 col-lg-1" >{% trans "Setup Duration" %}</label>
+                        <div class="col-md-3 col-lg-3">
+                            <div class="form-group col-md-3 col-lg-4">
+                              <input type="text" class="form-control" id="setup-duration">
+                            {% include "include/duration_picker_script.html" with selector="#setup-duration" %}
```

Comment:
Your widget class can use the `render()` function to render this file as its output so you can do something like `{{ field }}` inside the template.

Reply:
![duration](https://user-images.githubusercontent.com/28320272/114660955-27c9ed00-9cc4-11eb-99ea-ee441547bacb.png)
For some reason each field is displaying stacked under each other, I tried to put it into a row in different ways just like the [bootstrap example shows](https://github.com/koss-lebedev/bootstrap-duration-picker/blob/master/demo/example.html), but it hasn't worked yet. Im going to check the css and try more things.